pkg/kubernetes/client.js

```diff
@@ -252,6 +252,11 @@ define([
         var index = new HashIndex(262139);
 
         self.resourceVersion = null;
+        self.k8api = api;
+        self.POD = "Pod";
+        self.SERVICE = "Service";
+        self.RC = "ReplicationController";
+        self.NS = "Namespace";
```

Comment:
These should be global variables, and not per instance of KubernetesClient. It's too bad that not all our target browsers support javascript ```const``` so these would just be global (within the scope of client.js) definitions like:

```var POD = "Pod";```

For more about const see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const